php_phongo.c

```diff
@@ -632,10 +633,12 @@ bool phongo_execute_write(mongoc_client_t *client, const char *namespace, php_ph
 		return false;
 	}
 
+	opts = bson_new();
+
 	/* FIXME: Legacy way of specifying the writeConcern option into this function */
 	if (options && Z_TYPE_P(options) == IS_OBJECT && instanceof_function(Z_OBJCE_P(options), php_phongo_writeconcern_ce TSRMLS_CC)) {
 		zwriteConcern = options;
-	} else if (!phongo_execute_parse_options(client, server_id, options, PHONGO_COMMAND_WRITE, NULL, NULL, &zwriteConcern TSRMLS_CC)) {
+	} else if (!phongo_execute_parse_options(client, server_id, options, PHONGO_COMMAND_WRITE, opts, NULL, &zwriteConcern TSRMLS_CC)) {
```

Comment:
With no other changes to this function, `opts` are never used when executing the bulk write.

This isn't a concern for read preference and read concern, as those don't apply here. We also handle write concern explicitly below with a setter.

That said, it looks like we're missing an API in libmongoc to allow us to set a session on the bulk write. /cc @ajdavis